pdns/test-dnsdist_cc.cc

```diff
@@ -62,7 +62,7 @@ static void validateECS(const PacketBuffer& packet, const ComboAddress& expected
   uint16_t qtype;
```

Comment:
Looks like these tests only do the false/DoUDP case. It might be good to add more cases (this is an issue with the existing tests already)